### ISSUE TITLE
acs-commons-email-user does not have read permission on /conf folder

### ISSUE THREAD

**Issue by rajeevyadav2** [Adobe-Consulting-Services/acs-aem-commons#2199]
## Required Information

* [x] AEM Version, including Service Packs, Cumulative Fix Packs, etc: 6.5.3
* [x] ACS AEM Commons Version: 4.3.4
* [x] Reproducible on Latest? yes

### Expected Behavior

Email templates are stored under /conf folder and email-service user should have read permissions in . /conf folder so that it can load email templates.

### Actual Behavior

This service user does not have read permissions in /conf folder so email template is not loaded.

### Steps to Reproduce

1. Install acs package 4.3.4
2. Create an email template and keep it under /conf (subfolder)
3. Using EmailService, try to send the email which uses the template defined under /conf
4. Service throws exception as it is not able to load template due to lack of read permissions.

I tried giving /conf read permissions for acs-commons-email-service using ensure service configuration but it throws exception. Error says that it does not allow to update for AEM/ACS users.

### Links

_Links to related assets, e.g. content packages containing test components_

**Comment by dhardtke** [comment]
I can confirm this issue. For our project we created a `rep:policy` node for apps/settings/notification-templates/ourservlet that provides `acs-commons-email` with the necessary permissions:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:rep="internal"
          jcr:primaryType="rep:ACL">
    <allow
        jcr:primaryType="rep:GrantACE"
        rep:principalName="acs-commons-email-service"
        rep:privileges="{Name}[jcr:read]"/>
</jcr:root>
```

**Comment by HitmanInWis** [comment]
This will be resolved by https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/2218 shortly :)

**Comment by davidjgonzalez** [comment]
Fixed via 4.5.2
